### ISSUE TITLE
why do you use  “ratio_unsup=5.0” to weight the unsup_loss？

### ISSUE THREAD

**Issue by feimengjuan** [ildoonet/unsupervised-data-augmentation#12]
In the paper， the weight of unsup_loss is set to 1 to comput the whole loss，but in your code, the weight of unsup_loss is set as ratio_unsup=5.0. 
For example， https://github.com/ildoonet/unsupervised-data-augmentation/blob/master/train.py#L71 ，the code“loss += C.get()['ratio_unsup'] * torch.mean(loss_kldiv)”，in the wresnet28x2.yaml "ratio_unsup:5.0".

Could you please explain why?

When I set ratio_unsup=1.0， the final test Top-1 error rate is 11.63%.
